Modules/LearningSequence/classes/LSItems/LSItems.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * @author Daniel Weise <daniel.weise@concepts-and-training.de>
```

Comment:
"author" bitte in die Klassenberschreibung.

Reply:
ja